deb/debian/postinst

```diff
@@ -32,15 +32,56 @@ case "$1" in
           . /etc/default/percona-release
       fi
       directory="/etc/apt/sources.list.d"
+
       for file in $directory/percona*; do
         if [[ ! $file == *.bak ]]; then
+          # Skip files that contain 'signed-by='
+          if grep -q 'signed-by=' "$file"; then
```

Comment:
This causes a non-fatal error on apt installs:

```
grep: /etc/apt/sources.list.d/percona*: No such file or directory
ERROR: Invalid filename format: percona*
```

I'm not sure where this should be reported